### PR TITLE
Proofread docs: AB to Explosives

### PR DESCRIPTION
**Pull request by BullHorn7**
Continue proof-reading.
Fix more typos, improve grammar, remove redundancy.

### CODE REVIEW

File: documentation/missionmaker/modules.md [hunk 1]
```diff
@@ -11,34 +11,37 @@ parent: wiki
...
-Always enables advanced ballistics when high power optics are used.<br>
+2. **Enabled For Snipers (Boolean)**<br>
+Enables advanced ballistics for non local snipers (when using high power optics).<br>
 `Default value: Yes`
-3. **Disabled In FullAuto Mode (Boolean)**<br>
+3. **Enabled For Group Members (Boolean)**<br>
+Enables advanced ballistics for non local group members.<br>
+`Default value: No`
+4. **Enabled For Everyone (Boolean)**<br>
+Enables advanced ballistics for all non local players (Enabling this feature may degrade performance during heavy firefights in multiplayer).<br>
```
Comment (ulteq) [inline documentation/missionmaker/modules.md]: Maybe `Enabling` -> `enabling`

Comment (BullHorn7) [inline documentation/missionmaker/modules.md]: That's a good point. I'll check all the documents for parentheses that have a word starting with a capital letter where it shouldn't. Moment.

**Comment by ulteq** [comment]
We should merge https://github.com/acemod/ACE3/pull/1315 before we merge this.

**Comment by BullHorn7** [comment]
The module descriptions are correct even if #1315 isn't merged.

**Pull request merged**